tests/functional/data/updates.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from ..configuration.config import TEST_PATIENT_ID
+import re
+
+
+@dataclass
+class Update:
+    nhs_number: str
+    operation: str = 'replace'
+    path: str = 'gender'
+    _record: dict = None
+    _value: str = None
+    _etag: str = None
+
+    @property
+    def patches(self) -> dict:
+        patch = {
+            "op": f"{self.operation}",
+            "path": f"/{self.path}",
+            "value": f"{self.value}"
+        }
+        return {"patches": [patch]}
+
+    @property
+    def record(self):
+        return self._record
+
+    @property
+    def value(self):
+        return self._value
+
+    @property
+    def record_version(self):
+        return re.findall(r'\d+', self.etag)[0]
+
+    @property
+    def etag(self):
+        return self._etag
+
+    @record.setter
```

Comment:
should this be @record_version.setter ?

Reply:
No.

Reply:
There is a `record` property for the `Update` class. This is setting the property.